### DISCUSSION TITLE
Offcially support ElevenLabs TTS

### DISCUSSION THREAD

**Discussion by cyfyifanchen** [TEN-framework/ten-framework#90] [Announcements]
We are excited to announce that we officially support ElevenLabs Text-to-Speech (TTS). You can now experience high-quality, natural-sounding voice synthesis at https://theAstra.ai. Welcome to try it out and enhance your projects with the advanced capabilities of ElevenLabs TTS. ✨

**Comment by Innoglass** [comment]
May I know how I can use EleveLabs instead of Azure TTS?

**Reply by cyfyifanchen** [reply]
@Innoglass It's already available. There's an extension called `elevenlabs_tts`.

**Reply by Innoglass** [reply]
Is it I just provide elevelabs TTS key in the .env will do?

**Reply by cyfyifanchen** [reply]
Right.

**Comment by swloo618** [comment]
Does TEN Agent support AWS Speech to text?

**Reply by cyfyifanchen** [reply]
@swloo618 Yes, it does support Polly.

**Reply by Innoglass** [reply]
May I know how I can configure it?  Thanks

**Reply by cyfyifanchen** [reply]
Same as above.

**Reply by swloo618** [reply]
May I know if I just provide Agora ID, Agora Cert, AWS secret ket and access key, Elevelabs TTS Key, OpenAI key but no provide Azure TTS and STT key, will it able to run the Ten Agent?  Or I must provide  Azure STT key?  I try to run the ten agent but it not work.  I speak but nothing happen.

**Reply by cyfyifanchen** [reply]
@swloo618 Are you in our Discord channel? If you are, paste screenshots of log, let's take a look together.

**Reply by Innoglass** [reply]
Sorry, may I know where to get the log ya?

**Reply by cyfyifanchen** [reply]
@Innoglass @swloo618 come to our Discord, it's easier to ask questions and paste debug log. 
https://discord.gg/VnPftUzAMJ

**Reply by cyfyifanchen** [reply]
> Sorry, may I know where to get the log ya?

It's at the Terminal where you run `make run-server`.
